exercises/error-handling/error_handling_test.go

```diff
@@ -10,10 +10,10 @@ import (
 // (in all cases). The function should properly handle errors,
 // as defined by the expectations of this test suite.
 //
-// Also define an exported TestVersion with a value that matches
-// the internal testVersion here.
+// Also define an exported testVersion with a value that matches
+// the internal targetTestVersion here.
 
-const testVersion = 1
+const targetTestVersion = 2
```

Comment:
the fact that this is the only mention of `testVersion` in this file indicates to me that this is never being tested, probably good to put in a follow up